firmware/powerboard2/code/app/RTOS/FreeRTOSConfig.h

```diff
@@ -56,8 +56,8 @@
 #define configTICK_RATE_HZ				( ( TickType_t ) 1000 )
 #define configMAX_PRIORITIES			( 5 )
 #define configMINIMAL_STACK_SIZE		( ( unsigned short ) 100 )
-#define configTOTAL_HEAP_SIZE			( ( size_t ) ( 12 * 350 ) )
-#define configMAX_TASK_NAME_LEN			( 10 )
+#define configTOTAL_HEAP_SIZE			( ( size_t ) ( 16384 ) )
```

Comment:
Maybe add a comment why 16384 is important.

Reply:
Done.